.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,56 @@
 NewPipe contribution guidelines
 ===============================
 
-PLEASE READ THESE GUIDELINES CAREFULLY BEFORE ANY CONTRIBUTION!
-
 ## Crash reporting
 
-Do not report crashes in the GitHub issue tracker. NewPipe has an automated crash report system that will ask you to
-send a report via e-mail when a crash occurs. This contains all the data we need for debugging, and allows you to even
-add a comment to it. You'll see exactly what is sent, the system is 100% transparent.
+Report crashes in the automated crash report system of NewPipe.
+That way you get all data needed for debugging included in your bugreport for GitHub.
+You'll see exactly what is sent, add your comments, and then send it.
 
 ## Issue reporting/feature requests
 
-* Search the [existing issues](https://github.com/TeamNewPipe/NewPipe/issues) first to make sure your issue/feature
-hasn't been reported/requested before.
-* Check whether your issue/feature is already fixed/implemented.
-* Check if the issue still exists in the latest release/beta version.
-* If you are an Android/Java developer, you are always welcome to fix an issue or implement a feature yourself. PRs welcome!
-* We use English for development. Issues in other languages will be closed and ignored.
-* Please only add *one* issue at a time. Do not put multiple issues into one thread.
-* Follow the template! Issues or feature requests not matching the template might be closed.
+* **Already reported**? Browse the [existing issues](https://github.com/TeamNewPipe/NewPipe/issues) to make sure your issue/feature
+hasn't been reported/requested.
+* **Already fixed**? Check whether your issue/feature is already fixed/implemented.
+* **Still relevant**? Check if the issue still exists in the latest release/beta version.
+* **Can you fix it**? If you are an Android/Java developer, you are always welcome to fix an issue or implement a feature yourself. PRs welcome!
+* **Is it in English**? Issues in other languages will be ignored unless someone translates them.
+* **Is it one issue**?. Multiple issues require multiple reports, that can be linked.
```

Comment:
```suggestion
* **Is it one issue**?. Multiple issues require multiple reports so that we can track their status properly. 
```

Reply:
Trying to keep it short here, it doesn't need to be broken down into fundamentals.

Reply:
What about "allowing proper tracking"